### DISCUSSION TITLE
I miss the 'paint mode' and custom brushes, from enve =(

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#41] [Ideas]
I Know that friction is a fork to animate in svg to web. but i miss them =(

![image](https://github.com/friction2d/friction/assets/357488/c801c70c-200a-41ae-9e21-78a3bebd365e)
![image](https://github.com/friction2d/friction/assets/357488/ba898d27-e9c2-4b90-a418-23e9b588748a)
![image](https://github.com/friction2d/friction/assets/357488/561c6d1b-86f9-4972-8ac9-9974ee2ab8f9)

**Comment by rodlie** [comment]
I understand .... But Friction is not enve.

Friction is a tool to solve specific problems, and painting is not one of them.

One of the biggest issues in software development is feature/scope creep, and raster painting in Friction goes under feature/scope creep at the moment.

I don't know the justification for adding libmypaint in enve, but the implementation had it fair share of issues. It probably made sense for enve as it wanted to be a generic animation tool like synfig etc *(everything but the kitchen sink)*. Friction is not that, if you want to do generic animation with painting there are better (dedicated) alternatives available (every opensource graphics application under the sun has libmypaint).

I'm not saying that Friction will never get some kind of painting, but it must be planned. What are we trying to solve? and is the feature within the scope of the application? (currently no).

**Reply by 4l3m4oO** [reply]
All good. It was worth a shot =)

**Comment by alteregocom** [comment]
What are we trying to solve? I think it would be a good alternative for kids who want to animate but have a modest computer.
Drawing performance was very fast on old computers. Even faster than most free alternatives in my experience. The frame by frame black and white animation was very fast and fluid and the learning curve was short.
I understand the reason. Thank you for all your hard work.
![pleaseplease](https://github.com/friction2d/friction/assets/134562494/d1967d14-d16e-443a-8eb8-9b860f60cc51)

**Reply by tonypuglieso** [reply]
One reason I used paint in enve was to do effects when I have to draw them, and then add raster effects, like glow or blur. When one makes lightning or fire.

**Comment by tonypuglieso** [comment]
I know that friction is not designed for that, what I would use it for is more to use animations splash arts of illustrations that I animate in spine, and add the effects in Enve. Thinking if one day you could have a deform mesh (I don't think that will happen). It is a pity that even does not continue to be developed, to follow the line of exporting to video.

**Comment by ghost** [comment]
@4l3m4oO You could check out Pencil2D which is perhaps the most easy-to-use frame-by-frame animation tool https://www.pencil2d.org/

Or the following which are a bit more advanced
https://opentoonz.github.io/e/
https://krita.org/en/

**Reply by alteregocom** [reply]
I agree. Opentoonz performs well on old PCs and has the brilliant brushes from mypaint. But the learning curve is steep. Their forks are more user friendly but slightly slower in performance.
Wick Editor is a cute vector alternative.

**Reply by flurick** [reply]
@alteregocom I'm curious what performance did you experience slower in which fork? Enve feelt like it was so close to something very easy to use and at the same time powerful if needed with custom shaders etc, very exciting to see how Friction will develop (there is also still the option to use a group as a flipbook though :)

**Reply by tonypuglieso** [reply]
Hello 😁 I use kirta to draw and paint (in animation it is more limited). And opentoonz to do frame to frame animations, with a bit of compositing.

What is particular about Enve2D is a workflow more similar to vectorial After Effects, but you could also combine hand drawing, that made it super interesting, with good inkscape svg support, animating vector shapes. And then in Raster, you could put in a bezier line, a MyPaint brush texture. In addition to the workflow of keyframes.

For example, I can make a frame2frame animation in enve by combining the drawing I do with modifying the values ​​of transform and shear (with similar handling to blender), and then adding effects to it.

**Comment by alteregocom** [comment]
> @alteregocom I'm curious what performance did you experience slower in which fork? Enve feelt like it was so close to something very easy to use and at the same time powerful if needed with custom shaders etc, very exciting to see how Friction will develop (there is also still the option to use a group as a flipbook though :)

@flurick Tahoma 2D and Morevna Edition are slightly slower in my experience. It's almost imperceptible. But you can feel it when you switch between tools or scrubbing through the frames in the timeline. A shame because they bring substantial improvements. Amazings software.

OS: MX x86_64
Host: Satellite
CPU: AMD A8-6410 APU with AMD Radeon R5 Graphics (4) @ 2 
GPU: AMD ATI Radeon R4/R5 Graphics
Memory: 2011MiB / 6894MiB
